packages/mdc-shape/README.md

```diff
@@ -38,72 +35,27 @@ npm install @material/shape
 
 ## Basic Usage
 
-### HTML
-
-The markup for angled corners involves a container element, and one element per angled corner.
-
-The following example demonstrates angled corners applied to a MDC Button, but this technique may be applied to any
-unelevated component.
-
-```html
-<div class="mdc-shape-container my-shape-container">
-  <button class="mdc-button mdc-button--unelevated">Button</button>
-  <div class="mdc-shape-container__corner mdc-shape-container__corner--top-left"></div>
-  <div class="mdc-shape-container__corner mdc-shape-container__corner--top-right"></div>
-  <div class="mdc-shape-container__corner mdc-shape-container__corner--bottom-right"></div>
-  <div class="mdc-shape-container__corner mdc-shape-container__corner--bottom-left"></div>
-</div>
-```
-
-> **Note:** Each angled corner should have its own `<div>` element. Any corner that will not be cut (i.e. its size is 0)
-> does not need a dedicated element.
-
 ### Styles
 
-```scss
-@import "@material/shape/mdc-shape"; // The base shape styles need to be imported once in the page or application
-@import "@material/shape/mixins";
-
-.my-shape-container {
-  @include mdc-shape-angled-corner(#fff, 10px);
-}
-```
-
-## Variants
-
-### Outlined Angled Corners
-
-Outlined angled corners involve the same markup and styles/imports as above, with the addition of including a mixin for
-outline:
+The styles for applying custom shape to button component looks like this:
 
 ```scss
-.my-shape-container {
-  @include mdc-shape-angled-corner(#fff, 10px);
-  @include mdc-shape-angled-corner-outline(2px, blue);
+@import "@material/button/mixins";
+
+.my-custom-button {
+  @include mdc-button-shape-radius(pill);
```

Comment:
This doesn't actually exist in this branch, and I'm unsure OOC whether this is still how pill is expected to work?

Reply:
Updated README with relevant usage. PTAL.